pages/mesosphere/dcos/2.1/security/ent/tls-ssl/ar-custom/index.md

```diff
@@ -0,0 +1,242 @@
+---
+layout: layout.pug
+navigationTitle:  Configuring a Custom External Certificate
+title: Configuring a Custom External Certificate
+menuWeight: 50
+excerpt: Configuring DC/OS Enterprise to use a custom external certificate
+beta: true
+enterprise: true
+render: mustache
+model: /mesosphere/dcos/2.1/data.yml
+---
+<!-- The source repository for this topic is https://github.com/dcos/dcos-docs-site -->
+
+
+External access to a DC/OS Enterprise cluster goes through Admin Router. By default, the certificate presented by Admin Router is signed by the DC/OS Certificate Authority. The default DC/OS CA certificate is not trusted by default, and requires clients to manually configure trust in the CA (e.g. by accepting a pop-up browser dialog). A trusted CA certificate can be provided using [custom CA certificates](/mesosphere/dcos/2.1/security/ent/tls-ssl/ca-custom/). However, obtaining a suitable CA certificate can be difficult. Custom external certificates allow the cluster administrator to provide an easily obtained non-CA certificate and key that Admin Router will present to external connections.
+
+The benefits of using a custom external certificate for your DC/OS Enterprise cluster include:
+
+- only needing to obtain a standard certificate instead of an intermediate CA signing certificate, in order to make connections to the cluster trusted by the clients by default (i.e. browser-trusted certificates);
+- using a certificate with different properties for clients connecting to the cluster using different server names.
+
+# Contents
+- [Supported certificates](#supported-ca-certificates)
+- [Glossary](#glossary) for general definition of terms 
+- [Requirements](#requirements)
+- [Configuration parameter reference](#config-ref)
+- [Installation walkthrough](#installing-dcos-enterprise-with-a-custom-ca-certificate). 
+- [Example use cases](#example-use-cases) then provide example file contents for the custom external certificate configuration files for three popular use cases.
+
+# Supported certificates
+- Certificates with both RSA and ECC type keys are supported
+- Custom external certificates are only supported for a fresh installation of DC/OS Enterprise 2.1 or later. Older versions of DC/OS are not supported. It is possible to add a custom external certificate during upgrade to 2.1.
```

Comment:
The way I understand it is that Custom External Cert (let's just use CEC for short) cannot be added during the upgrade procedure to e.g. DC/OS 2.2 from DC/OS 2.1 - only the upgrade from older versions to the new version of fresh installs can give one the CEC.

Reply:
I understand why it would be a problem with a custom CA cert, but not why it is a problem for the external Admin Router cert. Why is it not possible to change this during an upgrade from 2.1 to 2.2?

Reply:
I am sorry - I was not precise. What I meant is that changing CEC during the upgrade from 2.1 to 2.2+ *IS* possible, but the text makes me think that it is not.

Reply:
I've simplified it down to 3 separate statements to avoid confusion.